### ISSUE TITLE
Hud, markers and depth meter dissapear

### ISSUE THREAD

**Issue by MaverickOne96** [Okabintaro/SubmersedVR#30]
After 5 to 10 min of playing hud, markers and depth meter dissapear. I see that disabeling and renabeling the option "Put survival meter on left wrist" temporarely fix it. 

Ps. This mod is great. Keep up the good work!

**Comment by SergejLogis** [comment]
This is not a bug, I believe, but rather an immersion feature. You can toggle HUD by touching watch area on your left wrist 🤓

**Comment by Okabintaro** [comment]
Yes, you are not the only one getting confused by this. I should somehow makes this more clear in the settings menu maybe.
